docs/book/user-guide/production-guide/connect-deployed-zenml.md

```diff
@@ -204,3 +204,6 @@ To keep things simple, we can summarize the steps:
 Every API key issued is a potential gateway to access your data, secrets and
 infrastructure. It's important to regularly rotate API keys and deactivate
 or delete service accounts and API keys that are no longer needed.
+
```

Comment:
There is an additional scarf in the middle of this page. One probably has to be removed.